docs/rst/user_guide/atmosphere/index.rst

```diff
@@ -1,3 +1,8 @@
+:orphan:
+
+.. Orphan tag suppresses 'WARNING: document isn't included in any toctree',
+   remove when document will be updated.
+
```

Comment:
I am working on updating the content of the atmosphere guide. I hope to have an updated version, maybe not complete, this week, and I shall add more content after the final meeting

Reply:
Sure; in the meantime, it's better IMO to hide the outdated content. You'll just have to revert that commit in addition to your current updates to restore access to the content.